store/migrations/1547744983_organizations.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+CREATE TABLE organization (
```

Comment:
AFAIU the couple `(provider, internal_id)` should be unique. In that case we could enforce it at db level by adding the corresponding constraint.

Reply:
I'm not sure, but from `go-kallax/README.md`:
>_Specifies the column has a unique constraint._
the `unique` constraint could be at column level, so composite uniqueness could be not supported by the ORM.

Reply:
I suppose that this sql script is executed in the DB as-is, so even if it's not supported by the ORM at least we can prevent "wrong data" being stored. But I don't know how the ORM will fail in case of a write that doesn't satisfy the constraint.

Reply:
I guess as any other write error?

Reply:
I don't know kallax so it may be just a generic write error as you said or it may crash in unpredictable ways (I'm thinking for example about nil pointer exception that won't let the user undestand what actually happened).

Reply:
I really hope it handle write errors

Reply:
+1 for constraint